riscv_isac/coverage.py

```diff
@@ -474,11 +684,8 @@ def compute_per_line(instr, mnemonic, commitvalue, cgf, xlen, addr_pairs,  sig_a
                     stats.stat2.append(_log + '\n\n')
                     stats.last_meta = [store_address, store_val, stats.covpt, stats.code_seq]
                 else:
-                    if len(stats.last_meta) != 0:
-                        _log = 'Last Coverpoint : ' + str(stats.last_meta[2]) + '\n'
-                        _log += 'Last Code Sequence : \n\t-' + '\n\t-'.join(stats.last_meta[3]) + '\n'
-                    else:
-                        _log = 'Signature Update without any coverpoints hit'
+                    _log = 'Last Coverpoint : ' + str(stats.last_meta[2]) + '\n'

```

Comment:
This change (5 lines) needs to revert back otherwise following error is seen saying "list index out of range"
![image](https://user-images.githubusercontent.com/63648044/125580309-d0e225b2-7d97-4e63-83c8-736e33dc93d9.png)

Reply:
@bilalsakhawat-10xe Please verify if [this](https://github.com/riscv/riscv-isac/pull/11) fixes your error.

Reply:
@pawks yes the error is fixed. Thanks